### DISCUSSION TITLE
HackBar破解

### DISCUSSION THREAD

**Discussion by giscus** [zu1k/blog#64] [Comments]
# HackBar破解

A boy dreaming of traveling around the world.<br>This blog is used to record my study, thinking and life.

https://zu1k.com/posts/security/web-security/hack-hackbar/

**Comment by LunarianCubed** [comment]
文中贴的官网已经变成了什么奇奇怪怪的东西。。。

**Reply by zu1k** [reply]
看起来不再维护了
